tests/modules/segmentation/graph_utils/feature_vector_segmentation/conftest.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+import pathlib
+import networkx as nx
+import h5py
+import tempfile
+import numpy as np
+
+
+@pytest.fixture
+def example_graph(tmpdir):
+    graph_path = pathlib.Path(tempfile.mkstemp(
+                                      dir=tmpdir,
+                                      prefix='graph_',
+                                      suffix='.pkl')[1])
+
+    rng = np.random.RandomState(5813)
+
+    graph = nx.Graph()
+
+    img = np.zeros((40, 40), dtype=int)
+    img[12:16, 4:7] = 1
+    img[25:32, 11:18] = 1
+    img[25:27, 15:18] = 0
+
+    for r0 in range(40):
+        for dr in (-1, 1):
+            r1 = r0+dr
+            if r1 < 0 or r1 >= 40:
+                continue
+            for c0 in range(40):
+                for dc in (-1, 1):
+                    c1 = c0 + dc
+                    if c1 < 0 or c1 >= 40:
+                        continue
+                    if img[r0, c0] > 0 and img[r1, c1] > 0:
+                        v = np.abs(rng.normal(0.5, 0.1))
+                    else:
+                        v = np.abs(rng.normal(0.1, 0.05))
+                    graph.add_edge((r0, c0), (r1, c1),
+                                   dummy_attribute=v)
+
+    nx.write_gpickle(graph, graph_path)
+    return graph_path
```

Comment:
I am a little surprised that this works, as the `tmpdir` instance should go out-of-scope upon this return, but perhaps your invocation of tempfile is somehow keeping it alive. I would recommend something like:
```
def myfixture(tmp_path):
    fpath = tmp_path / "something.pkl"
    ...
    yield fpath
```
then the tmp_path won't go away until the thing dependent on the fixture is done.

Reply:
Done